neo/private_key.go

```diff
@@ -21,6 +22,41 @@ type (
 	}
 )
 
+// NewPrivateKey creates a new random PrivateKey
+func NewPrivateKey() (*PrivateKey, error) {
+	b := make([]byte, 32)
+	_, err := rand.Read(b)
+	if err != nil {
+		return nil, err
+	}
+	return &PrivateKey{bytes: b}, nil
+}
+
+// NewWIFFromPrivateKey creates a WIF based on the private key
+func (p PrivateKey) NewWIFFromPrivateKey() (*string, error) {
```

Comment:
I would call this function `WIF()` as it follows the pattern of the other functions.

Reply:
Also please use `(string, error)` as the return types, and return `""` for the string if an error occurs.